### ISSUE TITLE
Merge kernel-cache & akmods repos for consistent kernel versioned builds

### ISSUE THREAD

**Issue by bsherman** [ublue-os/akmods#266]
Since these two repos have tightly coupled/dependent workflows, we should merge the workflows.

Blocks: #256

**Comment by piaste** [comment]
Hi @bsherman 

First, thanks so much for your work on the project. I've had five machines running on various uBlue flavours and I've been extremely happy with them for over a year :) 

Last week had to start maintaining a [simple custom Bazzite image](https://github.com/piaste/bazzite-nct6687d/blob/main/Containerfile) in order to re-include the NCT6687D module.

Right after #295 was merged, fixing this issue, [I was able to get it to run by using the `akmods-extra:bazzite-41` image alone](https://github.com/ublue-os/akmods/issues/300#issuecomment-2636597794) on top of `bazzite-stable`.

However, [the pipeline only worked once](https://github.com/piaste/bazzite-nct6687d/actions), and subsequent iterations ran into `kernel-uname-r` errors.

For example,  I see that the akmods-extra image was rebuilt 14 hours ago, yet 4 hours ago my automated build failed due to a kernel/kmod mismatch:

```
 Problem: conflicting requests
  - nothing provides kernel-uname-r = 6.12.12-206.bazzite.fc41.x86_64 needed by kmod-nct6687d-0.0.git.109.80dc037b-1.fc41.x86_64 from @commandline
Error: building at STEP "RUN rpm-ostree install /tmp/rpms/kmods/*nct6687*.rpm": while running runtime: exit status 1
```

I'm trying to figure out the correct way to fix this dependency. Is there a way to determine the correct `akmods-extra` tag to be used for whatever the latest `bazzite:stable` image is ? Or do I need to e.g. copy over the relevant parts from [fetch-kernel.sh script](https://github.com/ublue-os/akmods/blob/main/fetch-kernel.sh) and manually add the rpms? (But my understanding is that kernel packages _shouldn't_ be installed via `rpm-ostree`).

**Comment by bsherman** [comment]
> Hi [@bsherman](https://github.com/bsherman)
> 
> First, thanks so much for your work on the project. I've had five machines running on various uBlue flavours and I've been extremely happy with them for over a year :)

Thanks for hanging with us!

Could you please recreate this as a new issue in this repo?

![Image](https://github.com/user-attachments/assets/208565cc-9b0b-4879-8ec0-bce592387f98)

I'll answer there.
